compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/transform/CreateViewTransform.java

```diff
@@ -125,7 +126,8 @@ public String toString() {
    * @param <T> primitive view type
    */
   public static final class MultiView<T> implements Materializations.MultimapView<Void, T>, Serializable {
-    private final Iterable<T> iterable;
+    
+    private final transient Iterable<T> iterable;
```

Comment:
Please just remove the transient keyword and leave it untouched. This is rather a complex issue, so let's not resolve it.